react/react-training/src/app/components/AppRoutes/index.js

```diff
@@ -1,19 +1,39 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import connect from 'react-redux/es/connect/connect';
 import { Switch, Route } from 'react-router';
 import { BrowserRouter as Router } from 'react-router-dom';
 import Game from '@screens/Game';
 import Login from '@screens/Login';
 import { ROUTES } from '@constants/routes.js';
 
-export default class AppRoutes extends Component {
+import AuthRoute from './components/AuthRoute';
+
+class AppRoutes extends Component {
   render() {
+    const { userIsLogged } = this.props;
     return (
       <Router>
         <Switch>
-          <Route exact path={ROUTES.HOME} component={Login} />
+          <AuthRoute exact path={ROUTES.HOME} component={Login} userIsLogged={userIsLogged} />
```

Comment:
What would happen if I'm not logged in but I want to go to the path `/home` entering through the navigation bar? I think it allows me and it should not happen.

Reply:
Hola @WFolini 
Si el usuario ingresa a la url /home no sale nada de la app como puedes ver en la siguiente imagen. ¿Qué debería hacer?
<img width="833" alt="gabo s tic tac toe - home" src="https://user-images.githubusercontent.com/656353/48582269-08abb300-e8f2-11e8-8c2e-145c8e9cfe1d.png">